### PR TITLE
Update CNN3D model

### PR DESCRIPTION
**Pull request by chenzhang1709**
Enhance CNN_3D training pipeline:
1. Aggressive 3D augmentations in data_transforms
2. Volume grouping logic in image_loader
3. LR scheduler & gradient clipping in runner & optimizer
4. Residual Conv3D blocks + global pooling in model
5. Updates the results in main.ipynb

### CODE REVIEW

**Comment by willakins** [comment]
Copied all of your changes into main, but can't merge because of weird git issues with main.ipynb it seems
